lib/hammer_cli/main.rb

```diff
@@ -47,8 +47,8 @@ class MainCommand < AbstractCommand
       :context_target => :interactive
     option ["--no-headers"], :flag, _("Hide headers from output")
     option ["--csv"], :flag, _("Output as CSV (same as --output=csv)")
-    option ["--output"], "ADAPTER", _("Set output format. One of [%s]") %
-      HammerCLI::Output::Output.adapters.keys.join(', '),
+    option ["--output"], "ADAPTER", _("Set output format."),
+           format: HammerCLI::Options::Normalizers::Enum.new(HammerCLI::Output::Output::adapters.keys.map(&:to_s)),
```

Comment:
Prefer `Class.method` instead of `Class::method`.

Reply:
Thanks, I changed it.